tests/python_test_runner.py

```
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+"""
+Python-based test runner for Caramba.nvim Lua tests
+Parses and executes Lua test files without requiring Lua interpreter
+"""
+
+import os
+import re
+import sys
+import glob
+from typing import List, Dict, Any, Tuple
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)